cilantro_audit/view_audit_templates.py

```diff
@@ -0,0 +1,54 @@
+import kivy
+from kivy.app import App
+from mongoengine import connect
+from kivy.uix.button import Button
+from kivy.uix.screenmanager import Screen
+from kivy.uix.screenmanager import ScreenManager
+
+from cilantro_audit.constants import KIVY_REQUIRED_VERSION
+from cilantro_audit.constants import PROD_DB
+from cilantro_audit.home_page import HomePage
```

Comment:
W.R.T. Steven's comment on connecting this page using the flow `HomePage`->`AuditorPage`->`ViewAuditTemplates`, adding this page to the widgets in the `ScreenManager` object created in `HomePage` creates a circular import structure which cannot resolve. E.g. `HomePage` imports `ViewAuditTemplates`, and `ViewAuditTemplates` imports `HomePage`, meaning neither can be resolved.

I'm working on a fix that would work with the structure you are using for the page, and should hopefully be able to have a solution tomorrow.